install/kubernetes/helm/istio/values-istiotest.yaml

```diff
@@ -3,7 +3,7 @@
 global:
   nodePort: true
   proxy:
-    image: proxy_debug
+    image: proxyv2
```

Comment:
this is for the testing deploy, so should be using proxyv2_debug if there is a debug container for proxyv2.  If not, looks good as is - with the idea to add a debug version of proxy.

Reply:
we don't have proxyv2_debug. Hence the choice